step/src/main/java/com/google/sps/DatabaseHandler.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.NoSuchElementException;
+
+public final class DatabaseHandler {
+  
+  // TODO: replace with Datastore
+  private static Map<Long, User> idMap = new HashMap<>();
+
+  private DatabaseHandler() {
+
+  }
```

Comment:
What's this for?

Reply:
Both DatabaseHandler and MatchManager are final classes that contain only static methods. The idea is that you should be able to call the methods within these classes without having to actually make an instance of the class. Having a private constructor ensures that the class can't be instantiated or inherited anywhere.

Reply:
Ah I get it now, good thinking!